genus-library/src/main/scala/co/topl/genusLibrary/orientDb/GenusGraphMetadata.scala

```diff
@@ -77,7 +77,7 @@ package co.topl.genusLibrary.orientDb {
     /**
      * Schema for Address nodes
      */
-    private val addressVertexSchema: VertexSchema[TypedEvidence] =
+    implicit val addressVertexSchema: VertexSchema[TypedEvidence] =
```

Comment:
This and the other should not be public. We don't want to expose these details
```suggestion
    private[orientDb] implicit val addressVertexSchema: VertexSchema[TypedEvidence] =
```